salt/loader.py

```diff
@@ -1613,7 +1613,7 @@ def process_virtual(self, mod, module_name, virtual_func='__virtual__'):
                     error_reason = (
                         'Exception raised when processing __virtual__ function'
                         ' for {0}. Module will not be loaded: {1}'.format(
-                            module_name, exc))
+                            mod.__name__, exc))
```

Comment:
The difference with this is now we get a log message like:
```
15:26:18,561 [salt.loader:1617][ERROR   ] Exception raised when processing __virtual__ function for salt.loaded.int.module.gpg.
Module will not be loaded: unorderable types: str() < int()
```
Instead of:
```
15:26:18,561 [salt.loader:1617][ERROR   ] Exception raised when processing __virtual__ function for gpg.
Module will not be loaded: unorderable types: str() < int()
```